src/CMakeLists.txt

```diff
@@ -748,6 +748,8 @@ if (NOT WITH_SYSTEM_ROCKSDB)
     list(APPEND ROCKSDB_CMAKE_ARGS -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER})
   endif(WITH_CCACHE AND CCACHE_FOUND)
 
+  # Quick fix for lack of SSE4.2-accelerated CRC32 calculation in RocksDB
+  list(APPEND ROCKSDB_CMAKE_ARGS -DCMAKE_CXX_FLAGS=-msse4.2)
```

Comment:
we should at least check the building host's support of `-msse4.2`. what if we are building ceph on an aach64 machine? see `SIMDExt.cmake` and `HAVE_INTEL_SSE4_2`.